1.0.0-beta7-coreclr/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM debian:jessie
```

Comment:
@ahmetalpbalkan yeah, jessie is still on 0.10 while we need 1.x: https://packages.debian.org/jessie/libuv-dev